src/flutter.rs

```diff
@@ -826,6 +826,16 @@ impl InvokeUiSession for FlutterHandler {
         )
     }
 
+    fn set_multiple_user_session(&self, u_sids: String, u_names: String) {
+        self.push_event(
+            "set_multiple_user_session",
+            vec![
+                ("u_sids", &u_sids),
+                ("u_names", &u_names),
+            ],
```

Comment:
I guess sid and name are paired, you should not seperate them, and wast resource to double check sids.size == u_names.size. Please do as I suggested in the protobuf.